Mathlib/LinearAlgebra/PiTensorProduct.lean

```diff
@@ -747,6 +882,43 @@ end Tmul
 
 end Multilinear
 
+section
+
+open BigOperators Function
+
+variable [Fintype ι]
+
+variable (R ι)
+
+/-- The linear equivalence from the tensor product of the constant family with
+value `R` to `R`, given by multiplication of the entries.
+-/
+noncomputable def constantBaseRingEquiv : (⨂[R] (_ : ι), R) ≃ₗ[R] R := by
```

Comment:
I think this belongs in `RingTheory/PiTensorProduct`, as I believe it is an `AlgEquiv` with no stronger assumptions

Reply:
I have a version of this; I'll split it off to its own PR

Reply:
Should I take your :+1: as an agreement that the code there replaces this? If so, can you delete this, and then I'll merge the rest of this PR.

Reply:
Yes, I went at your PR, it looked good to me and I didn't see anything I'd want to change in it. I deleted the definition and lemma from this PR.